browser/lib/getTodoState.js

```diff
@@ -0,0 +1,25 @@
+export function getTodoState (content) {
```

Comment:
I changed that. And I also changed TodoProcess prop from `todoState` to `todoStatus`.
By the way, add transition for TodoProcess bar.